haystack/nodes/answer_generator/openai.py

```diff
@@ -99,6 +101,8 @@ def __init__(
             [PromptTemplate](https://docs.haystack.deepset.ai/docs/prompt_node#template-structure).
         :param context_join_str: The separation string used to join the input documents to create the context
             used by the PromptTemplate.
+        :param moderate_content: Whether to filter input and generated answers for potentially sensitive content
+            using the [OpenAI Moderation API](https://platform.openai.com/docs/guides/moderation)
```

Comment:
let's explicitly document here and also for the invocation layer that an empty list is returned in that case